.NET/Microsoft.Recognizers.Text.DateTime/Portuguese/Parsers/PortugueseSetParserConfiguration.cs

```diff
@@ -194,5 +194,7 @@ public bool GetMatchedUnitTimex(string text, out string timex)
         }
 
         public string WeekDayGroupMatchString(Match match) => SetHandler.WeekDayGroupMatchString(match);
+
+        public string ReplaceValueInTextWithFutTerm(string text, string value) => TasksModeSetHandler.ReplaceValueInTextWithFutTerm(text, value, DateTimeDefinitions.ThisTerms[0]);
```

Comment:
Like I mentioned before, it can be a list of terms. Not only the first in the list. Pass the list as parameter and loop it inside the method.

Reply:
TasksModeSetHandler.cs file method ReplaceValueInTextWithFutTerm now takes list as parameter.